config/env/production.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
       "https://hylo.com",
       "https://api.hylo.com",
       "https://staging.hylo.com",
+      "https://node1.hylo.com",
```

Comment:
is this a node backend replicant?

Reply:
its one of the urls we use with heroku for our backend yes. I'm not actually sure we need this, but was running into a weird issue accessing the backend for some API stuff and thought this might fix it. it's safe to do at least